seedfarmer/commands/_deployment_commands.py

```diff
@@ -273,15 +373,27 @@ def prime_target_accounts(
     ) as workers:
 
         def _prime_accounts(args: Dict[str, Any]) -> List[Any]:
+            target_account_id = args["account_id"]
+            target_region = args["region"]
+
             threading.current_thread().name = (
-                f"{threading.current_thread().name}-{args['account_id']}_{args['region']}"
+                f"{threading.current_thread().name}-{target_account_id}_{target_region}"
             ).replace("_", "-")
-            _logger.info("Priming Acccount %s in %s", args["account_id"], args["region"])
+            _logger.info("Priming Acccount %s in %s", target_account_id, target_region)
             seedkit_stack_outputs = commands.deploy_seedkit(**args)
             seedfarmer_bucket = commands.deploy_bucket_storage_stack(**args)
             seedkit_stack_outputs["SeedfarmerArtifactBucket"] = seedfarmer_bucket
             commands.deploy_managed_policy_stack(deployment_manifest=deployment_manifest, **args)
-            return [args["account_id"], args["region"], seedkit_stack_outputs]
+
+            deployment_role_name = create_generic_module_deployment_role(
+                account_id=target_account_id,
+                region=target_region,
+                deployment_manifest=deployment_manifest,
+                qualifier=qualifier,
+            )
+
+            seedkit_stack_outputs["DeploymentRoleName"] = deployment_role_name
```

Comment:
ummm, why add to the seedkit outputs?  Should this be hung on the targetAccount Mapping object?

Reply:
good call, on it